web-ts/tools/webpack.config.ts

```diff
@@ -0,0 +1,295 @@
+import {Configuration, DefinePlugin, LoaderOptionsPlugin, optimize} from "webpack";
+import {join, resolve} from 'path'
+import {CleanWebpackPlugin} from 'clean-webpack-plugin'
+import {readFileSync} from "fs";
+import {BundleAnalyzerPlugin} from 'webpack-bundle-analyzer'
+import MomentTimezoneDataPlugin from 'moment-timezone-data-webpack-plugin'
+import CopyPlugin from 'copy-webpack-plugin'
+import AssetsPlugin from 'assets-webpack-plugin'
+import HtmlWebpackPlugin from 'html-webpack-plugin'
+
+// const loadBuildConfig = require('build-configs').default
+// const MomentLocalesPlugin = require('moment-locales-webpack-plugin')
+// const babelConfig = require('../babel.config.js')
+// const translations = require('translations')
+// const { WEB, ANDROID, IOS } = require('build-configs')
+// reset the tsconfig to the default configuration
+delete process.env.TS_NODE_PROJECT
+const currentYear = new Date().getFullYear()
+
+const SHORT_COMMIT_SHA_LENGTH = 8
+
+// A first performance budget, which should be improved in the future: Maximum bundle size in Bytes; 2^20 = 1 MiB
+// eslint-disable-next-line no-magic-numbers
+const MAX_BUNDLE_SIZE = 1.64 * Math.pow(2, 20)
+// eslint-disable-next-line no-magic-numbers
+const MAX_ASSET_SIZE = 2.1 * Math.pow(2, 20)
+
+const readJson = (path: string) => JSON.parse(readFileSync(path, 'utf8'))
+
+const readVersionName = () => {
+    const versionFile = readJson(resolve(__dirname, '../../version.json'))
+    return versionFile.versionName
+}
+
+const generateManifest = (content: Buffer, buildConfigName: string) => {
+    // const manifest = JSON.parse(content.toString())
```

Comment:
We should add a TODO comment everywhere where we comment code out. Else we will miss important parts.